docs/user-documentation/2xfor1x.md

```diff
@@ -1,8 +1,8 @@
-#What's Different Between Islandora 1.x and 2.x
+#What's Different Between Islandora 1.x and CLAW
```

Comment:
Also, this whole section needs an update to mention Drupal 8. I'll take care of that once this purge is merged.